packages/mask/src/extension/popups/pages/Personas/ConnectedWallets/UI.tsx

```diff
@@ -18,6 +18,8 @@ const useStyles = makeStyles()(() => ({
         rowGap: 12,
         flex: 1,
         backgroundColor: '#F7F9FA',
+        overflow: 'scroll',
```

Comment:
```suggestion
        overflow: 'auto',
```

Don't always show scrollbar gutter